POLICY_LIST.md

```diff
@@ -52,6 +67,12 @@ Before adding a domain to the list, we continue to perform validation against th
 
 With that in mind, you can [queue your mail domain](https://starttls-everywhere.org/add-domain) for the STARTTLS policy list. Alternatively, you can send an email to [starttls-policy@eff.org](mailto:starttls-policy@eff.org) or [submit a pull request](https://github.com/EFForg/starttls-everywhere) to add your domain, though these other channels may take longer for your domain to be submitted.
 
+#### Which mode: `testing` to `enforce`?
+
+If your domain has a valid MTA-STS policy up, we'll use the mode indicated in the policy file.
+
+If your domain doesn't have a valid MTA-STS policy, your domain will be soft-queued in "testing" mode. Once your email domain has been on the list for at least two weeks, you can request it be upgraded to "enforce"-- send `starttls-policy@eff.org` an email.
```

Comment:
We should probably explain the difference between the `enforce` and `testing` modes and provide a recommendation.